src/language-js/print/statement.js

```diff
@@ -201,6 +201,18 @@ function shouldPrintSemicolonAfterClassProperty(node, nextNode) {
     }
   }
 
+  // Flow variance sigil +/- requires semi if there's no
+  // "declare" or "static" keyword before it.
+  if (
+    (nextNode.type === "ClassProperty" ||
```

Comment:
Even though the ast seems  not possible to be `PropertyDefinition`, but should we use `isClassProperty`? After all that's estree standard. Not important.

Reply:
Changed to use `isClassProperty` in 14c8be5f5da. Like you said, the `"PropertyDefinition"` case shouldn't come up, but I don't think it hurts to have it there either and the function call looks slightly clearer here.